### PR TITLE
Add collator infra docs for neumann and turing

### PR DESCRIPTION
**Pull request by irsal**

### CODE REVIEW

File: _docs/node-operator-requirements.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+Bare metal over cloud, especially when costs are a consideration to the node operator.
+- High internet connection (5 gigabits per second).
+- At least 8 CPU cores (fastest core speed).
+- At least 16GB of RAM.
+- At least 1TB of storage.
+- Monitoring and alerting in place (Prometheus, Grafana, the like).
+
+Please sync a few days before your intended collation / block production candidacy to sync the nodes. Once your node is synced, you will find that the block numbers are up to date with both the relay chain and the parachain. Check out the [Telemetry](https://telemetry.polkadot.io/#list/0x0f62b701fb12d02237a33b84818c11f621653d2b1614c777973babf4652b535d) site for more information.
+
+## Ports
```
Comment (irsal) [inline _docs/node-operator-requirements.md:19]: @nvengal - please fill in this section

File: _docs/node-operator-requirements.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+### Step X: Onboard your collator
+Once you're fully synced, you're ready to on-board as a collator. Proceed to this page to move forward and to start producing blocks. ![Collator On-boarding](./collators.md)
+
+----------------------------------------------------------------------------------------------------------------------------------
+## Turing Staging Network - Rococo Testnet
+Coming soon...
+----------------------------------------------------------------------------------------------------------------------------------
+## Turing Network - Kusama Parachain 
+### Step X: Grab the latest binary
```
Comment (irsal) [inline _docs/node-operator-requirements.md:92]: I'll add something about compiling here

File: _docs/node-operator-requirements.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  --node-key=NODE_KEY \
+  --collator \
+  --force-authoring \
+  --execution=wasm \
+  --state-cache-size=0 \
+  -- \
+  --chain=PATH_TO_RELAY_CHAIN_SPEC \
+  --execution=wasm \
+  --no-telemetry \
+  --bootnodes RELAY_CHAIN_BOOTNODE_P2P_ADDRESS \
```
Comment (irsal) [inline _docs/node-operator-requirements.md:139]: rm

File: _docs/node-operator-requirements.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
```
Comment (irsal) [inline _docs/node-operator-requirements.md:75]: rm

File: _docs/node-operator-requirements.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+Please sync a few days before your intended collation / block production candidacy to sync the nodes. Once your node is synced, you will find that the block numbers are up to date with both the relay chain and the parachain. Check out the [Telemetry](https://telemetry.polkadot.io/#list/0x0f62b701fb12d02237a33b84818c11f621653d2b1614c777973babf4652b535d) site for more information.
+
+## Ports
+@nikhil TODO
+Please ensure the following configuration for your node.
+- P2P port must be open to incoming traffic:
+    - Source: Any
+    - Destination: 30333, 30334 TCP
+- RPC port is closed down. (In other words, only you can do rpc calls.)
```
Comment (nvengal) [inline _docs/node-operator-requirements.md:25]: I removed this. By default as long as you're not adding the `--rpc-external` flag the node will not accept non-local rpc requests. One less thing node operators can worry about.

**Pull request merged**